VictoriaAlbert/VictoriaAlbert/DetailViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  DetailViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Marcel Chaucer on 11/10/16.
+//  Copyright © 2016 Marcel Chaucer. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+     var  detailObject: MuseumObject?
+
+    @IBOutlet weak var pictureOfObject: UIImageView!
+    
```

Comment:
You should only ever have one or two (at most) blank lines. Whatever spacing you choose to use, be consistent.